recipe/meta.yaml

```diff
@@ -1,5 +1,5 @@
 {% set name = "aiidalab" %}
-{% set version = "22.11.0" %}
+{% set version = "23.03.0" %}
```

Comment:
```suggestion
{% set version = "23.3.0" %}
```

Need to be like this I guess

Reply:
Yes, turns out pypi somehow normalizes the version.